pkg/sources/github/github_test.go

```diff
@@ -780,7 +780,6 @@ func TestGetRepoURLParts(t *testing.T) {
 	tests := []string{
 		"https://github.com/trufflesecurity/trufflehog.git",
 		"git+https://github.com/trufflesecurity/trufflehog.git",
-		//"git@github.com:trufflesecurity/trufflehog.git",
```

Comment:
IDK what's going on here, but I'm removing.

Reply:
That's fine. It's a remnant of different URLs style test cases.